src/main/webapp/pages/template/base.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html xmlns:th="http://www.thymeleaf.org"
       xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      th:lang="${#locale}">
+      th:lang="${#locale.language}">
```

Comment:
I think this is just leftover from a long time ago before you updated it. This also works to set the locale, but I reverted to your way in 88252bf6c41a18c21c8f8ff72e4e5e9e4adb14aa since we know it was working properly.